### ISSUE TITLE
tf2 branch no scene classification?

### ISSUE THREAD

**Issue by jyjyj** [TIBHannover/GeoEstimation#6]
Hi, as I recall from the original branch, there is a scene classification step before the geoestimation. In the tf2 branch, is this gone, or integrated into the geoestimation, and what would be the reasoning? Thanks!

**Comment by TIB-Visual-Analytics** [comment]
Hi, for simplification we omitted the code for the individual scenery networks (ISNs) that utilize scene classification in the tf2 branch. We decided to provide the base_M model as it achieves good results across all scene types and it only required to convert one model. 

If you want to train ISNs yourself follow these steps:
1. Prepare three datasets that contain images for natural, urban, or indoor scenes.
2. Train one network per dataset using the code in the tf2 branch
3. During inference predict the scene of the image and use the corresponding ISN for geolocation estimation. 

We hope we were able to help you.
